app/views/hub/notes/_note.html.erb

```diff
@@ -1,13 +1,14 @@
-<li class="note message message--<%= note.user == current_user ? "outgoing" : "incoming"%>" id="<%= "last-item" if local_assigns[:is_last_item] %>">
-  <div class="message__heading">
-    <span class="message__author">
-      <%= note.user.name %>
-    </span>
-    <span class="message__time">
-      <%= long_formatted_datetime(note.created_at) %>
-    </span>
+<div class="note <%= "system_note" if note.is_a? SystemNote %>" id="<%= "last-item" if local_assigns[:is_last_item] %>">
+  <div class="note__heading">
+    <div class="note__author">
+      <%= note&.user&.name %>
```

Comment:
should this be an ".author" method on the note, similar to how we handle contact records?